config/locales/en.yml

```diff
@@ -917,6 +918,7 @@ en:
         include_podcast_value: In some cases, you may wish to exclude micropayments from a private feed.
         itunes_category: Enter an alternate <a href="https://help.apple.com/itc/podcasts_connect/#/itc9267a2f12" target="_blank">Apple Podcasts category and subcategory</a>, differentiating this Feed from the Podcast.
         subtitle: Enter an alternate teaser
+        unique_guids: If checked, each episode will have a unique GUID in the RSS for this feed, based on the public GUID and the feed slug.
```

Comment:
I mean, all episodes have unique guids in their feeds, right?

Maybe something about how the guid will be _different_ than in the public RSS if you check this box?  (And that you usually wouldn't want that).

Reply:
text updated!